doc/examples/reconst_ivim.py

```diff
@@ -227,6 +226,89 @@ def plot_map(raw_data, variable, limits, filename):
 plot_map(ivimfit.D_star, "D*", (0, 0.01), "perfusion_coeff.png")
 plot_map(ivimfit.D, "D", (0, 0.001), "diffusion_coeff.png")
 
+"""
+
+Next, we will fit the same model with a more refined optimization process with
+`fit_method='VarPro'` (for "Variable Projection"). The VarPro computes the
+IVIM parameters using the MIX approach [Farooq16]_. This algorithm uses three
+different optimizers. It starts with a differential evolution
+algorithm and fits the parameters in the power of exponentials. Then the fitted
+parameters in the first step are utilized to make a linear convex problem.
+Using a convex optimization, the volume fractions are determined. The last step
+is non-linear least-squares fitting on all the parameters. The results of the first
+and second optimizers are utilized as the initial values for the last step of
+the algorithm.
+
+As opposed to the `'LM'` fitting method, this approach does not need to set any
+thresholds on the bvals to differentiate between the perfusion
+(pseudo-diffusion) and diffusion portions and fits the parameters simultaneously.
+Making use of the three step optimization mentioned above increases the
+convergence basin for fitting the multi-exponential functions of
+microstructure models. This method has been described in further detail in
+[Fadnavis19]_ and [Farooq16]_.
+"""
+
+ivimmodel_vp = IvimModel(gtab, fit_method='VarPro')
+
+t1 = time.time()
+ivimfit_vp = ivimmodel_vp.fit(data_slice)
+t2 = time.time()
+total = t2 - t1
```

Comment:
I think it would be good to print `total` or to remove `total, t1, t2`.  

It mainly depends on the goal so I'm good with one of this 2 options. Currently, It appears twice and you never really use this variable

Reply:
Good catch @skoudoro ... I think I will take it down for now, will print this in a later version after cythonizing a bit!